src/components/NavBar/NavBar.tsx

```diff
@@ -192,17 +192,29 @@ const NavBar: React.FC = () => {
 						>
 							Something
 						</MenuItem>
-						{checkUserRole(user, "validator") && (
-							<MenuItem
-								onClick={() => {
-									handleExploreClose();
-									navigate("/validator");
-								}}
-							>
-								Validator Panel
-							</MenuItem>
+						
+						{checkUserRole(user, "validator") && !checkUserRole(user, "admin") && (
```

Comment:
o kasıtlı olarak öyle, admin > validator olduğu için admin validator'un yapabildiklerini de yapıyor o yüzden eskisi gibi kalmalı burası